gettsim/parameters/wohngeld.yaml

```diff
@@ -1848,3 +1912,39 @@ heizkosten_zuschuss:
     5: 29.4
     jede_weitere_person: 3.6
   reference: Art. 1 G. v. 15.05.2020, BGBl I S. 1015.
+
```

Comment:
Yes, there was a change in form of the so-called permanent heating component, which was added (`heating_component`). The `heizkosten_zuschuss` refers to the 'amount for relief in heating costs due to CO2 pricing in euros', (see [here](https://www.bmwsb.bund.de/SharedDocs/gesetzgebungsverfahren/Webs/BMWSB/DE/Downloads/referentenentwuerfe/wohngeld-plus-gesetz-refe.pdf?__blob=publicationFile&v=1#page=6&zoom=auto,-82,477)) which was already implemented in 2021, so I would maybe suggest to rename it to something like `heating_cost_relief` so that you won't confuse it with the new component

Reply:
Ok, got it. It's a bit confusing since there is also the Klimakomponente and the [(temporary) Heizkostenzuschuss](https://www.bundesregierung.de/breg-de/aktuelles/heizkostenzuschuss-2130072). In light of the latter, we should definitely rename `heizkostenzuschuss`. 

Also, I just saw that it is defined on the monthly level, so we should add a `_m`.  Would be great if you can check the period also for the other variables in this file and add suffixes if they are not defined on the yearly level. I think we might have some other missing suffixes.